bin/list

```diff
@@ -172,16 +172,25 @@ const handler = async (req, res) => {
 
   related = decodeURIComponent(path.format(related))
 
+  let notFoundResponse
+
+  try {
+    const custom404Path = path.join(current, '/404.html')
+    notFoundResponse = fs.readFileSync(custom404Path, 'utf-8')
```

Comment:
Can you make this use `await`? Like [here](https://github.com/zeit/list/blob/master/bin/list#L211).